### DISCUSSION TITLE
Different layer height on one layer

### DISCUSSION THREAD

**Discussion by Seele86** [supermerill/SuperSlicer#2317] [Q&A]
Hi,
can I print defined parts of a layer with a different layer height?
For example I want to print this part with 0.25 mm layer height. Can I print the tooth row every half layer to geht more details? 

![image](https://user-images.githubusercontent.com/98843207/152633680-c2ec053f-849c-4eff-b6a4-d0ffdf306731.png)

**Comment by supermerill** [comment]
you can activate the variable layer height in the platter view, when you select an object.
The modified layer heights will be used for the whole object.
